utils/npm/node_modules/three.js/shim.js

```diff
@@ -0,0 +1,12 @@
+var THREE = require('three');
+
+console.warn( "WARNING: The 'three.js' npm package is deprecated in favor of the 'three' npm package, please upgrade.");
+
+if (typeof exports !== 'undefined') {
```

Comment:
Doesn't https://github.com/mrdoob/three.js/blob/dev/src/Three.js#L9-L17 do this already?